core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -34,6 +34,10 @@ class SearchService(
     with FileChangeListener
     with SLF4JLogging {
 
+  private[indexer] def applyBoost(path: String) =
+    (path startsWith config.rootDir.getAbsolutePath) &&
```

Comment:
sounds great, I'll track those down & move to checking against all of the module roots.